business_requirement_deliverable_report/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Deliverable Report Module',
+    'sumarry': 'Business Requirement Deliverable Report Module',
+    'version': '8.0.5.0.1',
```

Comment:
Actually it does somehow because we have been using the module internally with another version for 6 months